@ecomplus/storefront-app/src/components/html/EcCart.html

```diff
@@ -87,6 +87,17 @@
                 </span>
                 {{ i19checkout }}
               </a>
+
+              <a
+                class="cart__btn-checkout btn btn-block btn-outline-secondary d-md-none"
+                role="button"
+                href="/"
+              >
+                <span class="mr-1">
+                  <i class="fas fa-arrow-left mr-1"></i>
+                </span>
```

Comment:
```suggestion
                <i class="fas fa-arrow-left mr-1"></i>
```

Não precisa do span porque já tá o `mr-1` no ícone, se quiser você pode colocar `mr-2` pra uma margem maior (se a intenção era esta).